python/tvm/topi/arm_cpu/qnn.py

```diff
@@ -368,3 +389,139 @@ def kernel_ptr(buffer, c, offset=0):
 def schedule_qnn_depthwise_conv2d(_attrs, _outs, _target):
     """Schedule function for qnn.depthwise_conv2d."""
     return None
+
+
+def _make_unrolled_conv2d_primfunc(
```

Comment:
can you document the parameters here?

Reply:
Done, for both `_make_unrolled_conv2d_primfunc` and `_make_conv2d_primfunc`.